src/mapnik_feature.cpp

```diff
@@ -114,7 +114,7 @@ NAN_METHOD(Feature::fromJSON)
         }
         Feature* feat = new Feature(f);
         v8::Local<v8::Value> ext = Nan::New<v8::External>(feat);
-        Nan::MaybeLocal<v8::Object> maybe_local = Nan::NewInstance(Nan::New(constructor)->GetFunction(), 1, &ext);
+        v8::MaybeLocal<v8::Object> maybe_local = Nan::NewInstance(Nan::GetFunction(Nan::New(constructor)).ToLocalChecked(), 1, &ext);
```

Comment:
The return type from ` Nan::NewInstance` is `Nan::MaybeLocal<v8::Object>` I think, why the shift to `v8::MaybeLocal<v8::Object>`?

Reply:
Seems like an oversight (maybe I copied it from somewhere else in the code?). I'll fix those places where I've modified to use Nan::MaybeLocal instead.